Apps/phterraformcloud/terraformcloud.json

```diff
@@ -0,0 +1,2914 @@
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get run",
+            "description": "This endpoint is used for showing details of a specific run",
+            "type": "investigate",
+            "identifier": "get_run",
+            "read_only": true,
+            "parameters": {
+                "id": {
+                    "description": "The ID of the run to retrieve details",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "terraform run id"
+                    ],
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.id",
+                    "data_type": "string",
+                    "contains": [
+                        "terraform run id"
+                    ],
+                    "example_values": [
+                        "run-B45D8uRipKCtvpYd"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.relationships.configuration-version.data.type",
+                    "data_type": "string",
+                    "example_values": [
+                        "configuration-versions"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.relationships.configuration-version.data.id",
+                    "data_type": "string",
+                    "example_values": [
+                        "cv-ybgt5G3jpbW72V1s"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.relationships.configuration-version.links.related",
+                    "data_type": "string",
+                    "example_values": [
+                        "/api/v2/runs/run-B45D8uRipKCtvpYd/configuration-version"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.relationships.run-events.data.*.type",
+                    "data_type": "string",
+                    "example_values": [
+                        "run-events"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.relationships.run-events.data.*.id",
```

Comment:
Wondering if for some of these, we can add `contains`...

Reply:
We have checked and identify for workspace id contains and will add it for that wherever applicable. Usually, we introduce new custom contains if there any values from the datapaths are used in the action parameters of the same application.